### PR TITLE
[PAN-2845] Added eea_getPrivateTransaction

### PR DESCRIPTION
**Pull request by MadelineMurray**
Added eea_getPrivateTransaction

### CODE REVIEW

File: docs/Reference/Pantheon-API-Methods.md [hunk 1]
```diff
@@ -3771,6 +3771,54 @@ None
...
+using [`eth_getTransactionByHash`](#eth_gettransactionbyhash) with the transaction hash returned by 
+[`eea_sendRawTransaction`](#eea_sendrawtransaction) or [`eea_sendTransction`](https://docs.ethsigner.pegasys.tech/en/latest/Using-EthSigner/Using-EthSigner/#eea_sendtransaction). 
+
+**Parameters** 
+
+`data` - Value of `input` from the public transaction
+
+**Returns**  
+
+Object - [Transaction object](Pantheon-API-Objects.md#transaction-object), or `null` if not a participant in the private transaction.
```
Comment (iikirilov) [inline docs/Reference/Pantheon-API-Methods.md]: It is not a `Transaction object` as specified in `Pantheon-API-Objects.md`

I think we should specify a `Private Transaction object` and link it here

File: docs/Reference/Pantheon-API-Methods.md [hunk 2]
```diff
@@ -3771,6 +3771,56 @@ None
     The `EEA` API methods are not enabled by default for JSON-RPC. Use the [`--rpc-http-api`](Pantheon-CLI-Syntax.md#rpc-http-api) 
     or [`--rpc-ws-api`](Pantheon-CLI-Syntax.md#rpc-ws-api) options to enable the `EEA` API methods.
 
+### eea_getPrivateTransaction
+
+Returns the private transaction if you are a participant; otherwise, null. Get the public transaction
```
Comment (bgravenorst) [inline docs/Reference/Pantheon-API-Methods.md]: Minor suggestion, maybe make this a new paragraph. 
"To get the public transaction, use.."

File: docs/Reference/Pantheon-API-Objects.md [hunk 3]
```diff
@@ -65,6 +65,27 @@ Returned by [eth_getFilterChanges](Pantheon-API-Methods.md#eth_getfilterchanges)
...
+| **from**             | Data, 20&nbsp;bytes               | Address of the sender.                                                          |
+| **gas**              | Quantity                          | Gas provided by the sender.                                                     |
+| **gasPrice**         | Quantity                          | Gas price provided by the sender in Wei.                                        |
+| **hash**             | Data, 32&nbsp;bytes               | Hash of the transaction.                                                        |
+| **input**            | Data                              | Data to create or invoke contract.                                                                                 |
+| **nonce**            | Quantity                          | Number of transactions made by the sender to the privacy group before this one.                      |
+| **to**               | Data, 20&nbsp;bytes               | `null` if a contract creation transaction; otherwise, contract address             |
```
Comment (bgravenorst) [inline docs/Reference/Pantheon-API-Objects.md]: a few missing fullstops in the table

Comment (MadelineMurray) [reply docs/Reference/Pantheon-API-Objects.md]: That's consistent with the rest of the tables. We can revisit this on the wider basis when we look at the style guide

File: docs/Reference/Pantheon-API-Methods.md (hunk 2 above)
Comment (MadelineMurray) [reply docs/Reference/Pantheon-API-Methods.md]: Done

**Pull request merged**